u-boot/cpu/mips/ar7240/qca_dram.c

```diff
@@ -624,6 +629,11 @@ static inline void qca_dram_set_ddr_cfg2(u32 mem_cas,
 
 	/* Gate open latency = 2 * MEM_CAS */
 	tmp = 2 * mem_cas;
+#if (SOC_TYPE & QCA_AR933X_SOC)
```

Comment:
Good point, I'll try only setting CAS_LATENCY tomorrow and if it works, i'll remove the GATE_OPEN_LATENCY.